frontend/src/views/Lobby.tsx

```diff
@@ -255,7 +255,7 @@ function LobbyPage() {
       <br />
 
       {currentUser?.nickname === host?.nickname
-        ? <PrimaryButton onClick={handleStartGame} disabled={loading}>Start Game</PrimaryButton>
+        ? <PrimaryButton onClick={handleStartGame}>Start Game</PrimaryButton>
```

Comment:
nit: Re-add in this line that disables the start button when something is loading